server/services/zwavejs-ui/utils/convertToGladysDevice.js

```diff
@@ -55,6 +55,7 @@ const convertToGladysDevice = (serviceId, device) => {
   return {
     name: device.name,
     external_id: `zwavejs-ui:${device.id}`,
+    selector: `zwavejs-ui:${device.id}:${device.name}`,
```

Comment:
The device name can have spaces + not sure we want to tie the device to it's name? Maybe just the device.id ?